internal/containersuite/containerSuite.go

```diff
@@ -206,6 +205,7 @@ func (s *ContainerSuite) SetupSuite() {
 				"ZEEBE_BROKER_NETWORK_HOST":           "0.0.0.0",
 				"ZEEBE_BROKER_NETWORK_ADVERTISEDHOST": "0.0.0.0",
 			},
+			AlwaysPullImage: true,
```

Comment:
Not sure if this is really needed now with testContainers but I don't use it that tool frequently. 
With k8s it would not be needed.

Reply:
good point, I did this first and only then found the manual validation, will remove an try 👍